Core/RadioHandler.cpp

```diff
@@ -30,12 +30,6 @@ void RadioHandlerClass::OnDataPacket()
 		if (!run)
 			break;
 
-		if (fc != 0.0f)
-		{
-			std::unique_lock<std::mutex> lk(fc_mutex);
-			shift_limited_unroll_C_sse_inp_c((complexf*)buf, len, stateFineTune);
```

Comment:
why this get removed?

Reply:
I was unable to make decimator work as expected, so I partially removed it. Offset tuning was removed as well.

Reply:
Oscar may help and he is expert on that DSP code.